middlewares/osio/auth.go

```diff
@@ -16,9 +16,21 @@ const (
 	UserIDHeader  = "Impersonate-User"
 )
 
+const (
+	// service maps to token type, if not a service token then it maps to UserToken
+	CheToken  TokenType = "che"
+	UserToken TokenType = "user"
+)
+
+var TokenTypeMap = map[string]TokenType{
+	"rh-che": CheToken,
+}
+
+type TokenType string
+
```

Comment:
TokenType introduce to indicate is Token is service token (che, jenkins) if not it will be UserToken.